src/player/player.service.js

```diff
@@ -932,7 +932,7 @@ angular.module('ngGo.Player.Service', [
 
           //Call listener
           listener.apply(this, arguments);
-        });
+        }.bind(this));
```

Comment:
Because Angular would call this function with `null` context, we need to bind this function.